wiki/Beatmap_Editor_Guides/Adding_Custom_Hitsounds/de.md

```diff
@@ -0,0 +1,50 @@
+Eigene Hinsounds hinzufügen
+===========================
+
+Von dem Thread: [*Wie man einen Eigenen Hitsound hinzufügt* von: neonat](https://osu.ppy.sh/forum/p/3215699)
+
+Eigene Hitsounds
+---------------------------
+
+Als erstes werden einige Hitsounds benötigt.
+
+-   Man kann sie aus der [Custom Hitsound Library](Custom_Hitsound_Library "wikilink") bekommen.
```

Comment:
change the link to `[Custom Hitsound Library](/wiki/Beatmap_Editor_Guides/Custom_Hitsound_Library/)`
If `Custom Hitsound Library` is translatable, go ahead and translate it.